.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
- - [ ] This issue is about **specifically** the [git-scm.com](https://git-scm.com) website and is **not** an issue about
-     - [ ] Git (or its man pages), which should be raised with the [community](https://git-scm.com/community),
+- [ ] This issue is about **specifically** the [git-scm.com](https://git-scm.com) website and is **not** an issue about
+     - [ ] Git (or its man pages), which are located under https://git-scm.com/docs, should be raised with the [community](https://git-scm.com/community),
```

Comment:
Should this new phrase go inside the parenthetical with "man pages"? I.e.:

    Git (or its man pages, which are located under https://git-scm.com/docs), which should be raised...

Maybe "mirrored" would be better than "located", too, to make it clear that we are pulling in the content.